spyder/widgets/mixins.py

```diff
@@ -130,66 +129,144 @@ def _update_stylesheet(self, widget):
                 }}'''.format(name, background, border)
             widget.setStyleSheet(css + extra_css)
 
-    def _format_text(self, title, text, color, ellide=False):
+    def _get_inspect_shortcut(self):
+        """
+        Queries the application for the inspect shortcut defined by the
+        editor.
+        """
+        value = ''
+        app = QCoreApplication.instance()
+        # FIXME: This works, but there isprobably a cleaner way.
+        # This need to be defined on the console and on the editor
+        # For now only on the editor (eventually on the notebook)
+        shortcut_editor = getattr(app, 'shortcut_inspect_editor', None)
+        shortcut_console = getattr(app, 'shortcut_inspect_console', None)
+        shortcut = shortcut_editor or shortcut_console
+
+        if shortcut:
+            key_sequence = shortcut.data[0].key()
+            if key_sequence:
+                value = key_sequence.toString()
+            if sys.platform == "darwin":
+                value = value.replace('Ctrl', 'Cmd')
+        return value

```

Comment:
This is very very convoluted. I think we should use `CONF` for now and later decide how to better handle this kind of use cases.

Reply:
Fixed